packages/m/mitojs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@zyf2e/mitojs",
+  "description": "a SDK for monitoring browser errors",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/clouDr-f2e/mitojs.git"
+  },
+  "filename": "index.min.js",
+  "keywords": ["monitor", "sdk", "error", "broswer"],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/clouDr-f2e/mitojs.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
```

Comment:
There is no `dist` directory in your latest tagged version: https://github.com/clouDr-f2e/mitojs/tree/1.1.8